SecondTask/EntertainmentVenue.java

```diff
@@ -0,0 +1,34 @@
+package SecondTask;
+
+import java.util.Scanner;
+
+public class EntertainmentVenue {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Enter the customer's age:");
+        int age = scanner.nextInt();
+
+        if (age < 16) {
+            System.out.println("Access denied. Minimum age is 16.");
+            System.exit(2);
+        }
+        System.out.println("Is the customer a VIP? (true/false):");
+        boolean isVip = scanner.nextBoolean();
+
+        int entranceFee = 5000;
+//i am adding some changes for gitHub
+
+        System.out.println("Enter the amount for order:");
+        double orderAmount = scanner.nextDouble();
+        if (isVip) {
+            double discount = (orderAmount * 15)/100;
+            orderAmount = orderAmount - discount;
+        }
+        double totalAmountSpent = entranceFee + orderAmount;
```

Comment:
entrance fee exists for non-vip clients only. Based on that, do some corrections.